CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 - Added distributed tracing using open telemetry APIs for table stored procedure function in `DataFrame`:
   - _execute_and_get_query_id
+- Allow df.plot() and series.plot() to be called, materializing the data into the local client
```

Comment:
nit: trailing . at end of sentence.
Also I'd suggest putting ticks ` around the APIs in the log, so it looks prettier. I.e.,

`df.plot()` vs. df.plot()